node/node_bridge_test.go

```diff
@@ -42,100 +22,14 @@ func TestBridgeLifecycle(t *testing.T) {
 	ctx, cancel := context.WithCancel(context.Background())
 	t.Cleanup(cancel)
 
-	err = node.Start(ctx)
-	require.NoError(t, err)
-
-	ctx, cancel = context.WithCancel(context.Background())
-	t.Cleanup(cancel)
-
-	err = node.Stop(ctx)
-	require.NoError(t, err)
-}
-
-// TestBridge_P2P_Streams tests the ability for Bridge nodes to communicate
-// directly with each other via libp2p streams.
-func TestBridge_P2P_Streams(t *testing.T) {
-	store := MockStore(t, DefaultConfig(Bridge))
-	node, err := New(Bridge, store)
-	require.NoError(t, err)
-	require.NotNil(t, node)
-	require.NotNil(t, node.Host)
-
-	peerConf := DefaultConfig(Bridge)
-	// modify address to be different
-	peerConf.P2P.ListenAddresses = []string{
-		"/ip4/0.0.0.0/tcp/2124",
-		"/ip6/::/tcp/2124",
-	}
-	store = MockStore(t, peerConf)
-	peer, err := New(Bridge, store)
-	require.NoError(t, err)
-	require.NotNil(t, peer)
-	require.NotNil(t, peer.Host)
-
-	nodeCtx, nodeCtxCancel := context.WithCancel(context.Background())
-	t.Cleanup(nodeCtxCancel)
-
-	// connect node to peer
-	peerAddrID := host.InfoFromHost(peer.Host)
-	err = node.Host.Connect(nodeCtx, *peerAddrID)
-	require.NoError(t, err)
-	// create handler to read from conn on peer side
-	protocolID := protocol.ID("test")
-	peer.Host.SetStreamHandler(protocolID, func(stream network.Stream) {
-		buf := make([]byte, 5)
-		_, err := stream.Read(buf)
-		require.NoError(t, err)
-		require.Equal(t, "hello", string(buf))
-	})
-	// open a stream between node and peer
-	stream, err := node.Host.NewStream(nodeCtx, peer.Host.ID(), protocolID)
-	require.NoError(t, err)
-	// write hello to peer
-	_, err = stream.Write([]byte("hello"))
-	require.NoError(t, err)
-
-	// stop the connection
-	require.NoError(t, stream.Close())
-}
-
-func TestBridge_WithRemoteCore(t *testing.T) {
-	// TODO(@Wondertan): Fix core
-	t.Skip("Skip until we fix core")
```

Comment:
There were some flakiness or something idk, but here as well Swamp tests this for us